src/tools/fuzzing.h

```diff
@@ -2946,7 +2946,7 @@ class TranslateToFuzzReader {
   Expression* makeRefIsNull(Type type) {
```

Comment:
Do you expect to change this function name eventually? If so, it might be worth a TODO.